dask/array/core.py

```diff
@@ -2660,6 +2660,10 @@ def bincount(x, weights=None, minlength=None):
 
     return Array(dsk, name, chunks, dtype)
 
+@wraps(np.digitize)
+def digitize(a, bins, right=False):
+    func = partial(np.digitize, bins=bins, right=right)
+    return elemwise(func, a)
```

Comment:
Sorry but I don't think there is map_partitions on the array interface. There seems to be one on DataFrame.